ngraph/src/ngraph/op/proposal.hpp

```diff
@@ -83,6 +83,35 @@ namespace ngraph
                 ProposalAttrs m_attrs;
             };
         }
+
+        namespace v4
+        {
+            class NGRAPH_API Proposal : public op::v0::Proposal
+            {
+            public:
+                static constexpr NodeTypeInfo type_info{"Proposal", 4};
+                const NodeTypeInfo& get_type_info() const override { return type_info; }
+                Proposal() = default;
+                /// \brief Constructs a Proposal operation
+                ///
+                /// \param class_probs     Class probability scores
+                /// \param bbox_deltas     Prediction of bounding box deltas
+                /// \param image_shape     Shape of image
+                /// \param attrs           Proposal op attributes
+                Proposal(const Output<Node>& class_probs,
+                         const Output<Node>& bbox_deltas,
+                         const Output<Node>& image_shape,
+                         const ProposalAttrs& attrs);
+
+                virtual bool visit_attributes(AttributeVisitor& visitor) override;
+                virtual std::shared_ptr<Node>
+                    clone_with_new_inputs(const OutputVector& new_args) const override;
+                const ProposalAttrs& get_attrs() const { return m_attrs; }
+            private:
+                ProposalAttrs m_attrs;
+            };
+        }
+
         using v0::Proposal;
```

Comment:
@GlebKazantaev, should we update to `v4::Proposal`?

Reply:
I propose to remove this legacy code. We shouldn't upgrade it to new version.

Reply:
I would suggest not to have this default aliases to ops in op namespace. So I agree with @ilyachur to remove this.

Reply:
Yes. I will remove them.